docker/.env

```
@@ -260,3 +260,63 @@ UV_GUARDRAILS_DB_NAME=guardrails
+AM_CERTS_GRPC_CLIENT_CA_CERTS=/ssl/certs/certificates/ca-cert.pem
+AM_CERTS_RELEASE_TAG=latest
+
+## OpenBao PKI Config
+AM_CERTS_OPENBAO_HOST=http://certs-openbao:8200
+AM_CERTS_OPENBAO_APP_ROLE=absmach
+AM_CERTS_OPENBAO_APP_SECRET=absmach
+AM_CERTS_OPENBAO_SECRET_ID_TTL=87600h
+AM_CERTS_OPENBAO_NAMESPACE=
+AM_CERTS_OPENBAO_PKI_PATH=pki
+AM_CERTS_OPENBAO_ROLE=absmach
+AM_CERTS_OPENBAO_PKI_CA_CN=Abstract Machines Certificate Authority
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AM_CERTS_OPENBAO_PKI_CA_CN key should go before the AM_CERTS_OPENBAO_PKI_PATH key

(UnorderedKey)